app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :posts
   has_many :likes
   has_many :comments
+
+  def recent_posts
+    posts.first(3)
```

Comment:
[OPTIONAL] It's a good idea to use `Post.order(created_at: :desc).limit(3)` to retrieve the latest posts. This is because its more dependable and its not affected by deletion or updates in the database records.